<ui version="4.0" >
 <class>AddBookmarks</class>
 <widget class="QDialog" name="AddBookmarks" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" >
   <item row="4" column="0" colspan="3" >
    <widget class="QLineEdit" name="pathlineEdit" />
   </item>
   <item row="5" column="1" >
    <widget class="QPushButton" name="okButton" >
     <property name="text" >
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2" >
    <widget class="QPushButton" name="cancelButton" >
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" >
    <spacer name="horizontalSpacer" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QLineEdit" name="titlelineEdit" />
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Title:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
